components/eco-bank/EcoBankDetail.vue

```diff
@@ -248,6 +250,7 @@ const props = defineProps<{
   website: string;
   rating: string;
   bankFeatures: BankFeature[];
+  countries: { code: string; }[];
```

Comment:
Why is this prop added? I don't see it being used anywhere in this PR 😅

Reply:
ah I originally was using it but then found the helper, will remove